gettsim/data/wohngeld_neu.yaml

```diff
@@ -177,6 +177,7 @@ koeffizienten:
           b: 0.000377
           c: 0.0000173
     2001-01-01:
+      reference: Art. 5 G. v. 22.12.1999 BGBl. I S. 2671
       value:
```

Comment:
I think we should drop the `value` level here. Really only makes sense for scalars (maybe we should even call it `scalar`?), else it adds an unnecessary level of nesting.

Reply:
But we should add a note that the keys refer to household sizes. Trivial, but useful for casual observers.

Reply:
I am not sure. When we read the yaml file it is a lot easier to select `value`. Or is there a easy way, which I don't see?

Reply:
I wouldn't add this below the date and instead a note at the very beginning of koeffizenten.

Reply:
> I wouldn't add this below the date and instead a note at the very beginning of koeffizenten.

Fair enough.

Reply:
> I am not sure. When we read the yaml file it is a lot easier to select `value`. Or is there a easy way, which I don't see?

Why would it? Instead of `[date]["value"][x]`, you just do `[date][x]` ? 

It probably makes sense to discard any non-integers in the transformation, though. As in #140:

```python
keys = sorted(key for key in list_of_dicts.keys() if type(key) == int)
```

Reply:
Yes. So its tradeoff between nesting and discarding keys. I just found nesting more readable in the yaml file and also, as mentioned before, easier to process. But I do not have strong preferences on this, thus I will just change!